docs/fieldnorm/user_feedback.md

```diff
@@ -0,0 +1,21 @@
+# Laboratory patients, user, and personnel feedback
+
+Bioinformaticians develop data analysis workflows, tools and (web) applications that are used by different groups of laboratory analists and specialists, from hereafter refered as 'end users'. These end users will experience events where unexpected, incorrect or occasionally no results are produced, due to bugs which are caused by either errors, flaws or faults in the design, development, or operation of the software developed by the bioinformaticians. In addition, end users may come up with requests and suggestions for new features / functionality to further improve or expand the scope of existing software. 
+
+The bioinformatics team therefore needs to setup a system through which both reports of issues / bugs and requests can be submitted, traced and processed. Rather than using a single procedure / system to handle both issues and new requests or feedback, different procedures may be set up to process and handle them independently in a different way. When the issues have been handled and actions have been taken, results need to be communicated back to the end users. 
```

Comment:
I feel like "having a system" already encapsulates the possibility for having multiple procedures. I feel like it may only add to the confusion for mentioning the possibility